Detectors/EMCAL/workflow/src/AnalysisClusterSpec.cxx

```diff
@@ -0,0 +1,163 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include <gsl/span>
+
+#include "FairLogger.h"
+
+#include "DataFormatsEMCAL/Digit.h"
+#include "DataFormatsEMCAL/Cluster.h"
+#include "DataFormatsEMCAL/EMCALBlockHeader.h"
+#include "DataFormatsEMCAL/TriggerRecord.h"
+#include "EMCALWorkflow/AnalysisClusterSpec.h"
+#include "Framework/ControlService.h"
+#include "EMCALBase/Geometry.h"
+#include <TGeoManager.h>
+
+using namespace o2::emcal::reco_workflow;
+
+template <class InputType>
+void AnalysisClusterSpec<InputType>::init(framework::InitContext& ctx)
+{
+  LOG(DEBUG) << "[EMCALClusterizer - init] Initialize clusterizer ...";
+
+  // FIXME: Placeholder configuration -> get config from CCDB object
+  double timeCut = 10000, timeMin = 0, timeMax = 10000, gradientCut = 0.03, thresholdSeedEnergy = 0.1, thresholdCellEnergy = 0.05;
+  bool doEnergyGradientCut = true;
+
+  // FIXME: Hardcoded for run II run
+  // Get default geometry object if not yet set
+  mGeometry = Geometry::GetInstanceFromRunNumber(223409);
+  if (!mGeometry) {
+    LOG(ERROR) << "Failure accessing geometry";
+  }
+  gGeoManager->Import("/Users/hadi/Clusterizer/geometry.root");
```

Comment:
@shahor02 We need to access the geometry during the clusterization in order to get the rotation and misalignment matrices. In AliRoot we usually took them from the OCDB. Is there something similar foreseen for O2 (CCDB)? Otherwise could we just add the geometry file like a run2 OADB file?

In any case hard-coded path needs to be changed.

Reply:
Hi @mfasDa 
Eventually we will have it in the CCDB, but at the moment it is changing too often. Currently we use the geometry in the working dir. created during the simulation, see e.g. https://github.com/AliceO2Group/AliceO2/blob/27ece718a8940bc30dcc5f4d1ca1dd8b8f8f3b89/Detectors/ITSMFT/ITS/workflow/src/ClustererSpec.cxx#L41

Reply:
Hi @shahor02, @mfasDa 
Yes, this method worked. Thanks.